.github/README.md

```
@@ -29,25 +1,326 @@
+# CaptchaService
 
 [![Documentation][documentation-shield]][documentation]
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "documentation-shield"

(MD052, reference-links-images)